app/decorators/member_decorator.rb

```diff
@@ -0,0 +1,64 @@
+require 'dry-initializer'
+
+class MemberDecorator
+  extend Dry::Initializer
+
+  param :member
+  param :view_context
+
+  alias_method :v, :view_context
+
+  delegate :user, :member_uid, :active?, to: :member
+  delegate :phone, :alt_phone, :username, to: :user
+
+  def manager?
+    !!member.manager
+  end
+
+  def row_css_class
+    'bg-danger' unless active?
+  end
+
+  def inactive_icon
+    v.tag(:span, class: %w[glyphicon glyphicon-time]) unless active?
```

Comment:
@rewritten I see this produces a different html compared to the original code: an anchor inside the span (called along with `link_to_self`). I think this is because this line returns a self-closing span. It looks like this:

![captura de pantalla 2018-05-12 a la s 18 01 52](https://user-images.githubusercontent.com/576701/39959366-a286f356-5610-11e8-90cf-4efbfaf3c955.png)

We can fix that by using the `glyph` helper:
    
    v.glyph('time') unless active?

I just tried it locally and it looks fine:

![captura de pantalla 2018-05-12 a la s 18 18 42](https://user-images.githubusercontent.com/576701/39959383-f2f6b89e-5610-11e8-8c0a-9636e68e93af.png)

Reply:
Or `content_tag` with empty content